### DISCUSSION TITLE
Botpress Integration Guide

### DISCUSSION THREAD

**Discussion by trcalgaro** [open-wa/wa-automate-nodejs#2647] [General]
Hi

Would it be possible to create a BotPress integration tutorial?

How does the integration work, does the user browse the boot created and receive the response in wa?

**Comment by smashah** [comment]
Here's the original instructions:

https://github.com/open-wa/wa-automate-nodejs/issues/1932#issuecomment-885373639

```bash
> npx @open-wa/wa-automate@latest -b http://localhost:3000/api/v1/bots/test/converse    
```

**Comment by smashah** [comment]
https://github.com/open-wa/wa-automate-nodejs/issues/2072#issuecomment-968351928
